tasks/utilities/task_organism_defaults.wdl

```diff
@@ -0,0 +1,249 @@
+version 1.0
```

Comment:
Do these need to be tasks run with a VM?

Reply:
Probably not but miniwdl check was sad otherwise. We definitely don't need a VM for this, though